src/app/dance-moves/dance-move-edit/dance-move-edit.component.html

```diff
@@ -1 +1,120 @@
-<p>dance-move-edit works!</p>
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            <form
+                [formGroup]="danceMoveForm"
+                (ngSubmit)="onSubmit()">
+                <div class="row">
+                    <div
+                        class="col-xs-12"
+                        style="margin-bottom: 6%;
+                        margin-top: 3%;">
```

Comment:
Adding styles to the template is not good practice. Try adding a css class if you can :)